Building from src code-generation fails

# description:
building from src with the makefile, failed in 2 points at moment.
1) the go.sum failed saying the checksum was outdated. ( by this error i just removed the file, then worked)
2) the deepcopy failed ( i have installed last one)
i have tried this out in 2 differents OS-Leap15 Computers

Using $GOPATH
~/go/src/github.com/kubic-project/kubic-init> 

# howto
when building from source with 
```
make
>>> Updating deepcopy files in pkg/config/
sh -c GO111MODULE=off deepcopy-gen -O zz_generated.deepcopy -i ./... -h ../../hack/boilerplate.go.txt
F1108 22:18:13.821865    6534 deepcopy.go:866] Hit an unsupported type invalid type.
pkg/config/config.go:18: running "sh": exit status 255
make: *** [Makefile:103: pkg/config/zz_generated.deepcopy.go] Error 1

go version go1.11.2 linux/amd64

```

As stated in desc, i was thinking i had an unstable env. but i tried it with a new Env on other Computer and i get now same error

--- a/Makefile
+++ b/Makefile
@@ -91,6 +91,7 @@ print-version:
 $(DEEPCOPY_GENERATOR):
 	@echo ">>> Getting deepcopy-gen (for $(DEEPCOPY_GENERATOR))"
 	@$(GO_NOMOD) get k8s.io/code-generator/cmd/deepcopy-gen
+	@$(GO_NOMOD) get -u k8s.io/apimachinery
 
 define _CREATE_DEEPCOPY_TARGET
 $(1): $(DEEPCOPY_GENERATOR) $(shell grep -l "//go:generate" $(dir $1)/*)

--- a/Makefile
+++ b/Makefile
@@ -91,7 +91,7 @@ print-version:
 $(DEEPCOPY_GENERATOR):
 	@echo ">>> Getting deepcopy-gen (for $(DEEPCOPY_GENERATOR))"
 	@$(GO_NOMOD) get k8s.io/code-generator/cmd/deepcopy-gen
-	@$(GO_NOMOD) get -u k8s.io/apimachinery
+	@$(GO_NOMOD) get k8s.io/apimachinery
 
 define _CREATE_DEEPCOPY_TARGET
 $(1): $(DEEPCOPY_GENERATOR) $(shell grep -l "//go:generate" $(dir $1)/*)

